src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -14,6 +14,9 @@
           = @pagetitle
           %span.badge.ml-1{ class: "badge-#{request_badge_color(@bs_request.state)}" }
             = @bs_request.state
+            - if @bs_request.superseded_by.present?
```

Comment:
I decided against using a helper or a view component here since this is really simple. If it somehow grows in complexity in the future, we could extract this somewhere else.